cmd/find.go

```diff
@@ -49,21 +49,16 @@ func findMain(cmd *cobra.Command, args []string) exitCode {
 		return exitCode{1}
 	}
 
-	apiPath, err := client.APIKeyFromPath(path)
-	if err != nil {
-		cmdutil.ErrPrintf("%v\n", err)
-		return exitCode{1}
-	}
 	conn := client.ForUNIXSocket(config.Socket)
 
-	e, err := conn.Info(apiPath)
+	e, err := conn.Info(path)
 	if err != nil {
 		cmdutil.ErrPrintf("%v\n", err)
 		return exitCode{1}
 	}
 	entries := []apitypes.Entry{e}
 	if e.Supports(plugin.ListAction) {
-		children, err := conn.List(apiPath)
+		children, err := conn.List(path)
```

Comment:
One nice thing about this PR is that `wash find` now prints absolute paths.

Reply:
It shouldn't though. In what circumstance does it make sense to print absolute paths?

Reply:
Nevermind, brainfart on my part.

Reply:
Fixed.